qualang_tools/addons/calibration/README.md

```diff
@@ -0,0 +1,167 @@
+# Calibration API
+
+## Introduction
+
+This module allows to easily perform most of the standard single qubit calibrations from a single python file.
+
+The following calibrations are currently available:
+* [raw_traces](#raw-traces): acquires the raw and averaged traces from the two inputs of the OPX.
+* [time_of_flight](#time-of-flight): measures the time of flight and offsets of the two inputs of the OPX.
+* [resonator_spectroscopy](#resonator-spectroscopy): performs the resonator spectroscopy.
+* [Rabi](#rabi-measurement): performs a Rabi experiment where the pulse amplitude, duration and frequency can be scanned.
+* [T1](#t1-measurement): measures the qubit T1 by scanning the time after the application of a $\pi/2$ pulse.
+* [Ramsey](#ramsey-measurement): performs a Ramsey measurement where the idle time and pulse frequency can be scanned.
+
+## Usage Example
+1. First the user needs to specify the relevant configuration parameters that will be used for the calibrations.
+Are resuired the _qubit_ and _readout-resonator_ elements and operations, the _integration weights_ and OPX inputs (defined as elements outputs).
+An optional _flux_line_ element and operation can be specified in the case of flux tunable transmon to find the flus soft spot with the resonator spectroscopy.
+Several options are also available for customizing the plots. All of them are written in the snippet below.
+
+2. Then the calibration API is initialized with these parameters. The standard configuration also needs to be provided.
+3. The sequence of calibrations needs to be set using the `.set_` method. All the avainlable calibrations and setting methods are detailed in the *Available calibrations* section below. Note that several calibrations can be set and executed sequentially.
+4. One needs to open a quantum machine manager with the correct IP address.
```

Comment:
You're changing the tense between the lines. Sometimes it's indirect: "the sequence of calibrations needs to be set" and sometimes it's direct: "One needs to open a quantum machine manager".
I would change it to: "A quantum machine**s** manager needs to be opened"